diff --git a/Makefile.am b/Makefile.am
index 9767349..b059523 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -15,12 +15,12 @@ libgrit_la_SOURCES	= libgrit/cprs.cpp libgrit/cprs_huff.cpp libgrit/cprs_lz.cpp
 			libgrit/grit_prep.cpp libgrit/grit_shared.cpp libgrit/grit_xp.cpp \
 			libgrit/logger.cpp libgrit/pathfun.cpp \
 			libgrit/cprs.h libgrit/grit.h libgrit/grit_core.h \
-			libgrit/logger.h libgrit/pathfun.h
+			libgrit/logger.h libgrit/pathfun.h extlib/fi.cpp extlib/fi.h
 
 libgrit_la_CPPFLAGS	=	-I$(top_srcdir)/cldib
 
 grit_SOURCES	=	srcgrit/cli.cpp srcgrit/grit_main.cpp srcgrit/cli.h extlib/fi.cpp extlib/fi.h
 grit_LDADD	=	libgrit.la libcldib.la $(FREEIMAGE_LIBS)
-grit_CPPFLAGS	=	-I$(top_srcdir)/cldib -I$(top_srcdir)/libgrit -I$(top_srcdir)/extlib
+grit_CPPFLAGS	=	-I$(top_srcdir)/cldib -I$(top_srcdir)/libgrit -I$(top_srcdir)/extlib -std=c++14
 
 EXTRA_DIST = autogen.sh
diff --git a/configure.ac b/configure.ac
index b3646b8..0131887 100644
--- a/configure.ac
+++ b/configure.ac
@@ -10,6 +10,7 @@ AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([subdir-objects])
 LT_INIT([disable-shared])
 
+CXXFLAGS="$CXXFLAGS -std=c++14"
 # Checks for programs.
 AC_PROG_CXX
 AC_PROG_INSTALL
